src/MainWindow.vala

```diff
@@ -23,6 +23,7 @@ public class Installer.MainWindow : Hdy.Window {
     private AccountView account_view;
     private LanguageView language_view;
     private KeyboardLayoutView keyboard_layout_view;
+    private ThirdPartySoftwareView third_party_software_view;
```

Comment:
Most software in elementary OS is by third parties, maybe proprietary_software_view? Although I'm not even sure if that's the perfect term for it.

Reply:
Maybe keep it simple and just call it **Software**?

We could also set other options here
E.g. for developers and directly install `elementary-sdk`.